content/reference-transaction-format.md

```diff
@@ -730,6 +788,58 @@ Transactions of the TrustSet type support additional values in the [`Flags` fiel
 | tfClearFreeze | 0x00200000 | 2097152 | [Unfreeze](concept-freeze.html) the trustline. |
 
 
+
+## SignerListSet ##
+[[Source]<br>](https://github.com/ripple/rippled/blob/ef511282709a6a0721b504c6b7703f9de3eecf38/src/ripple/app/tx/impl/SetSignerList.cpp "Source")
+
+The SignerListSet transaction creates, modifies, or removes a list of signers that can be used to [multi-sign](#multi-signing) a transaction.
```

Comment:
Here's just a thought for your consideration.  A SignerListSet can't really _modify_ a list of signers, but it can _replace_ a list.  I don't know whether that is worth bringing up, but you could consider replacing "modifies" with "replaces" in this sentence.  Your call.